tests/autoyast/login.pm

```diff
@@ -19,7 +19,9 @@
 use strict;
 use base 'y2logsstep';
 use testapi;
-use ipmi_backend_utils;
+# TODO: is_remote_backend we need to ensure if for autoyast we want this or
+# this also applies for other backends
+use Utils::Backends qw (use_ssh_serial_console is_remote_backend);
```

Comment:
no space between `qw` and `(`

Reply:
Yeah, this is iirc what travis complains about